sass/module/_snackbar copy.scss

```diff
@@ -0,0 +1,97 @@
+@mixin applyPosition (
```

Comment:
What is this file? seems like a duplicate of _snackbar.scss?

Reply:
You're right, it's duplicated, I've deleted it